tpl/bootstrap.php

```diff
@@ -66,11 +66,11 @@
 endif;
 if ($MARKDOWN) :
 ?>
-		<script type="text/javascript" data-cfasync="false" src="js/showdown-1.9.1.js" integrity="sha512-nRri7kqh3iRLdHbhtjfe8w9eAQPmt+ubH5U88UZyKbz6O9Q0q4haaXF0krOUclKmRJou/kKZYulgBHvHXPqOvg==" crossorigin="anonymous"></script>
+		<script type="text/javascript" data-cfasync="false" src="js/showdown-2.0.0.js" integrity="sha512-UB9jpMTOJLSnVzePuqlSGT34G70wEGqtIWabMeAh+Drnj4/uQ8rFkFn1zkN9vkWp/7nA51U2LmP23H5MJvBXsw==" crossorigin="anonymous"></script>
```

Comment:
Manually checked and compared with https://github.com/showdownjs/showdown/releases/tag/2.0.0 -> https://github.com/showdownjs/showdown/releases/download/2.0.0/showdown.min.js

You removed the source mapping, okay, fair enough...

```
$  diff -s showdown-2.0.0.js showdown.min.js 
2a3
> //# sourceMappingURL=showdown.min.js.map
```

Reply:
Yes, I always do that, because they cause errors in the console and an failed extra request - at least when you've got the dev console open. Since we don't shop the map (as I at least never had to debug inside any of these 3rd party libraries) I want to avoid causing extra requests or misleading errors.